lib/compat/wordpress-6.2/additional-filters.php

```diff
@@ -0,0 +1,46 @@
+<?php
+function gutenberg_add_template_to_default_template_title( $label ) {
+	global $post_type, $post;
+	if ( $label !== __( 'Default template' ) || empty( $post ) || empty( $post_type ) || ! current_theme_supports( 'block-templates' ) ) {
+		return $label;
+	}
+
+	$block_template;
+	if ( $post_type === 'page' ) {
+		$templates = array();
+		if ( ! empty( $post->post_name ) ) {
+			$templates[] = "page-{$post->post_name}.php";
```

Comment:
```suggestion
			$templates[] = "page-{$post->post_name}";
```

I think we should drop the `php` suffix for block templates.

Reply:
The feedback was addressed 👍